tests/unit/remote/test_s3.py

```diff
@@ -99,3 +101,11 @@ def test_copy_preserve_etag_across_buckets(remote):
     to_etag = RemoteS3.get_etag(s3, "another", "foo")
 
     assert from_etag == to_etag
+
+
+def makedirs(remote):
```

Comment:
```suggestion
def test_makedirs(remote):
```
otherwise this test won't run 🙂